src/MagnumPlugins/AssimpImporter/CMakeLists.txt

```diff
@@ -70,6 +70,8 @@ endif()
 # temporary CMakeList.txt created by try_compile:
 # https://github.com/Kitware/CMake/blob/v3.4.0/Source/cmCoreTryCompile.cxx#L313
 file(WRITE ${CMAKE_CURRENT_BINARY_DIR}/checkAssimpVersionLanguageOverride.cmake "enable_language(C)")
+# For some reason, this also needs C to be enabled in this scope
+enable_language(C)
```

Comment:
Sigh. Gotta admit I don't like this part at all :/ I'm wondering if I could expand the above static library workaround to not need to link to the `assimp` target ever, sidestepping the problem with a C99 feature enabled on it.